packages/garbo/src/resources/aprilingband.ts

```diff
@@ -12,12 +12,13 @@ const instruments: {
   {
     instrument: "Apriling band quad tom",
     value: () =>
-      globalOptions.prefs.valueOfFreeFight +
-      0.02 * garboValue($item`spice melange`),
+      (globalOptions.prefs.valueOfFreeFight +
+        0.02 * garboValue($item`spice melange`)) *
+      3,
   },
   {
     instrument: "Apriling band saxophone",
-    value: () => getBestLuckyAdventure().value(),
+    value: () => getBestLuckyAdventure().value() * 3,
```

Comment:
Strip the * 3s from this and from the piccolo, and correct the piccolo value to 40 (that one's on me)

Reply:
(and from the quad tom ofc)